test/com/google/javascript/jscomp/CommandLineRunnerTest.java

```diff
@@ -2532,6 +2532,34 @@ public void testWebpackModuleIds() throws IOException {
                 + "\"}]");
   }
 
+  @Test
+  public void testInstrumentCodeByLine() {
+    args.add("--instrument_code=LINE");
+
+    test(
+            "function foo(){ const answerToAll = 42; }",
+            "self.window||(self.window=self,self.window.top=self);var __jscov=window.top.__jscov||"
```

Comment:
For newer unit tests we've been using the `lines` helper function instead of string concatenation to create long strings.

Could you replace this test case and the one below with 

```
lines(
    "self.window||(self.window=self,self.window.top=self);var __jscov=window.top.__jscov||",
    "(window.top.__jscov={fileNames:[],instrumentedLines:[],executedLines:[]}),",
[ etc. ]
)